health/micro-ui/web/packages/modules/campaign-manager/src/components/GenerateExcelJs.js

```
@@ -0,0 +1,380 @@
+      link.click();
+      document.body.removeChild(link);
+      window.URL.revokeObjectURL(url);
+
+    } catch (error) {
+      // Download failed - error handled silently
+    }
+  };
+
+  return (
+    <div style={{ display: "none" }}>
+      <button ref={inputRef} onClick={handleExport}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)